pyroengine/vision.py

```diff
@@ -47,7 +47,7 @@ def preprocess_image(self, pil_img: Image.Image) -> np.ndarray:
         """
 
         # Resizing
-        img = pil_img.resize(self.cfg["input_shape"][-2:], Image.BILINEAR)
+        img = pil_img.resize(self.cfg["input_shape"][-2:][::-1], Image.BILINEAR)
```

Comment:
Please use Image.Resampling.BILINEAR instead, Image.BILINEAR is [deprecated](https://pillow.readthedocs.io/en/stable/deprecations.html)

Reply:
This change would require also to update the version specifiers for Pillow, so I suggest doing it in another PR :+1: 

(either putting <9.1.0 in the specifier, or going for your change + >=X.Y.Z, where X.Y.Z is the first release where Resampling become supported)

Reply:
OK let's stay like this then